test/acceptance/log.provider.test.ts

```diff
@@ -0,0 +1,90 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: loopback-next-extension-starter
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import { Client, createClientForApp, sinon } from '@loopback/testlab';
+import { inject } from '@loopback/context';
+import {
+  Application,
+  get,
+  param,
+  SequenceHandler,
+  FindRoute,
+  ParseParams,
+  InvokeMethod,
+  Send,
+  Reject,
+  CoreBindings,
+  ParsedRequest,
+  ServerResponse,
+} from '@loopback/core';
+import { LogComponent, ExtensionStarterBindings, LogFn } from '../../src';
```

Comment:
You should not import from `src` files, as that's not how the module is used at runtime. You should import the main `index.js` which loads Node.js-specific version. I am actually surprised this code is working for you without ts-node